tardis/plasma/properties/ion_population.py

```diff
@@ -275,6 +270,13 @@ class IonNumberDensity(ProcessingPlasmaProperty):
     for the electron densities are not similar to within the threshold
     value, a new guess for the value of the electron density is chosen
     and the process is repeated.
+    
+    Attributes
+    ----------
+    ion_number_density : pandas.DataFrame, dtype float
+        Index atom number, ion number. Columns zones.
+    electron_densities : numpy.ndarray
+        dtype float
```

Comment:
```suggestion
    electron_densities : numpy.ndarray, dtype float
```
dtype float is not description its additional info that numpy ndarray has elements of float type

Reply:
I wasn't sure, as Wolfgang said that it should be moved to be below, so I have a question on his comment asking about it.